openpype/hosts/maya/plugins/publish/extract_gpu_cache.py

```diff
@@ -0,0 +1,45 @@
+import os
+
+from maya import cmds
+
+from openpype.pipeline import publish
+from openpype.hosts.maya.api.lib import (
+    maintained_selection, attribute_values, delete_after
+)
+
+
+class ExtractGPUCache(publish.Extractor):
```

Comment:
Probably best to allow this extractor to be enabled/disabled in settings.

Reply:
Good point. Got a load of other settings to implement later.